rules/autoconsent/remarkable.com.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "remarkable.com",
+    "vendorUrl": "https://www.remarkable.com/",
+    "cosmetic": false,
+    "runContext": {
+        "urlPattern": "^https://(www\\.)?remarkable\\.com/"
+    },
+    "prehideSelectors": ["[data-cy=cookie-consent-banner-info-only],[data-cy=cookie-consent-banner]"],
+    "detectCmp": [
+        {
+            "exists": "[data-cy=cookie-consent-banner-info-only],[data-cy=cookie-consent-banner]"
```

Comment:
These selectors look wrong to me (when accessing from DE):
![Screenshot 2025-03-07 at 15 24 21](https://github.com/user-attachments/assets/4ebbad5e-c407-47af-a7cd-204c2ea86235)

Reply:
they have changed the implementation, I updated the rule